src/HtmlUtils.js

```diff
@@ -44,8 +45,16 @@ var sanitizeHtmlParams = {
     allowedSchemesByTag: {},
     
     transformTags: { // custom to matrix
-        // add blank targets to all hyperlinks
-        'a': sanitizeHtml.simpleTransform('a', { target: '_blank'} )
+        // add blank targets to all hyperlinks except vector URLs
+        'a': function(tagName, attribs) {
+            var m = attribs.href.match(linkifyMatrix.VECTOR_URL_PATTERN);
+            if (m) {
+                return { tagName: 'a', attribs: { href: attribs.href } };
+            }
+            else {
+                return { tagName: 'a', attribs: { href: attribs.href, target: '_blank'} };
```

Comment:
the `allowedAttributes` list allows `name`, but we're dropping it here. Is that deliberate?